dcmgr/lib/dcmgr/configurations/hva.rb

```diff
@@ -277,6 +277,9 @@ def verbose_openflow(v)
 
       param :lxc_log_level, :default => :error
 
+      # network interface down script for lxc (ovs)
+      param :lxc_ifdown, :default => '/opt/axsh/wakame-vdc/scripts/lxc-ifdown'
```

Comment:
I think the name of this parameter might be a little confusing since it's only executed when `edge_networking` is openvnet. The current name suggests that it's execute for every LXC instance, regardless of `edge_networking`.